README.md

```diff
@@ -158,6 +158,12 @@ License: By requesting and using the Container OS Image for Windows containers,
 - [Window Server Core](https://hub.docker.com/r/microsoft/windowsservercore/)
 - [Nano Server](https://hub.docker.com/r/microsoft/nanoserver/)
 
+### Telemetry
+
+By default, PowerShell collects the OS description and the version of PowerShell (equivalent to `$PSVersionTable.OS` and `$PSVersionTable.GitCommitId`) using [Application Insights](https://azure.microsoft.com/en-us/services/application-insights/).
```

Comment:
Are we going with `$PSVersionTable.OS` or `[System.Runtime.InteropServices.RuntimeInformation]::OSDescription` (or are they same)? Relevant to my presentation in a couple days...